docs_user/modules/proc_adopting-image-service-with-object-storage-backend.adoc

```diff
@@ -73,8 +71,9 @@ spec:
             - storage
```

Comment:
(Hackfest feedback) Does the indentation look correct for “spec: type: LoadBalancer”?

Reply:
I remember I fixed that as part of https://issues.redhat.com/browse/OSPRH-7862